typescript/cli/src/deploy/warp.ts

```diff
@@ -818,6 +798,34 @@ function transformIsmConfigForDisplay(ismConfig: IsmConfig): any[] {
   }
 }
 
+/**
+ * Submits a set of transactions to the specified chain and outputs transaction receipts
+ */
+async function submitWarpApplyTransactions(
+  chain: string,
+  params: WarpApplyParams,
+  transactions: AnnotatedEV5Transaction[],
+) {
+  const submitter: TxSubmitterBuilder<ProtocolType> =
+    await getWarpApplySubmitter({
+      chain,
+      context: params.context,
+      strategyUrl: params.strategyUrl,
+    });
+
+  const transactionReceipts = await submitter.submit(...transactions);
+  if (transactionReceipts) {
+    const receiptPath = `${params.receiptsDir}/${chain}-${submitter.txSubmitterType}-receipts.json`;
```

Comment:
maybe you could include a timestamp or something in the filename to avoid overwriting

Reply:
goodea